### ISSUE TITLE
Support for if/else is broken

### ISSUE THREAD

**Issue by judepereira** [asyncy/platform-engine#165]
With the change in the Storyscript tree, if/else is broken. Story to reproduce this:
```coffee
if foo == true
  log msg
else if foo == false
  log msg
else
  log msg
```

Simulate this with a unit test to reproduce the issue. Also, the existing if/else test case needs to be reworked upon.

**Comment by judepereira** [comment]
@kego1992 Can you look into this one?

**Comment by kego1992** [comment]
Yes, Will get it done.

**Comment by judepereira** [comment]
Fixed with #181.
